src/main/resources/META-INF/plugin.xml

```diff
@@ -93,6 +99,12 @@
                 <!--                <override-text place="GitMacheteToolbar"/>-->
             </action>
 
+            <action id="GitMachete.ResetCurrentBranchAction"
```

Comment:
I'd also rename the action classes (including the base) to make it clear that the branches are reset to their remote [tracking branch].